scripts/visual-baselines.js

```diff
@@ -0,0 +1,71 @@
+/* jshint node: true */
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+const rimraf = require('rimraf');
+const logger = require('@blackbaud/skyux-logger');
+
+const {
+  exec,
+  dirHasChanges
+} = require('./utils');
+
+const baselineScreenshotsDir = 'screenshots-baseline';
+const tempDir = '.skypagesvisualbaselinetemp';
```

Comment:
This is introducing another temporary folder that could leak into a commit.  What are your thoughts on establishing a root-level tmp folder, then slowly start to migrate everything to under that folder, including updating the `.gitignores` for that folder?

I don't necessarily think it has to happen in this PR, just something would be benefit us in the future and stop our SPAs from having to add more folders to `.gitIgnore`.

Reply:
That's a good point. Here's an issue to track that effort: https://github.com/blackbaud/skyux2/issues/1966

Regarding this specific use-case, only the CI will be running this step, so the temp directory shouldn't be generated locally for our consumers.